com.unity.render-pipelines.universal/Runtime/PostProcessPasses.cs

```diff
@@ -30,28 +45,40 @@ internal struct PostProcessPasses : IDisposable
 
         public bool isCreated { get => m_CurrentPostProcessData != null; }
 
-        public PostProcessPasses(PostProcessData rendererPostProcessData, Material blitMaterial)
+        private PostProcessPasses(PostProcessData rendererPostProcessData)
         {
             m_ColorGradingLutPass = null;
             m_PostProcessPass = null;
             m_FinalPostProcessPass = null;
             m_CurrentPostProcessData = null;
+            m_BlitMaterial = null;
 
             m_AfterPostProcessColor = null;
             m_ColorGradingLut = null;
-
             m_RendererPostProcessData = rendererPostProcessData;
-            m_BlitMaterial = blitMaterial;
+        }
 
+        public PostProcessPasses(PostProcessData rendererPostProcessData, ref PostProcessParams postProcessParams) : this(rendererPostProcessData)
```

Comment:
Refactored this to have a struct interface for cleaner future param additions.

Feedback? C# is a bit clumsy for the backcompat coding, better ideas?